src/unit_tests/analysisd/test_dbsync.c

```diff
@@ -188,6 +188,33 @@ static int setup_DispatchDBSync(void **state) {
     return 0;
 }
 
+static int setup_DispatchDBSync2(void **state) {
```

Comment:
Comment: this setup already defines a message that can't be changed.
So we are removing many different invalid components, and leaving one single test that verifies this invalid component doesn't work.

It might not be worth the effort, but for future reference, the setup should contain all the common test case code, and each UT should verify slight variations of the payload.

Reply:
Considering https://github.com/wazuh/wazuh/issues/27891#issuecomment-2637909751, I agree that there were methods called that don't trigger the error message.

Reply:
Ok, I got it after the discussion we had. I didn't touch the already existent tests and setups, I create a new one without initializing the payload, what is in charge of each test case. Take a look